packages/wrangler/src/dev.tsx

```diff
@@ -374,6 +374,14 @@ function useEsbuild(props: {
         ...(jsxFactory && { jsxFactory }),
         ...(jsxFragment && { jsxFragment }),
         external: ["__STATIC_CONTENT_MANIFEST"],
+        conditions: [
+          "worker",
+          "browser",
+          "development",
```

Comment:
let's remove the `development`/`production` conditions, we need to figure out the environment story separately

Reply:
done